languages/javascript/config.json

```diff
@@ -97,6 +97,12 @@
         "uuid": "5aa39e89-c601-4a66-ab72-5d8512d69e02",
         "concepts": ["closures"],
         "prerequisites": ["arrays", "booleans"]
+      },
+      {
+        "slug": "errors",
+        "uuid": "be0614cf-2958-4cc4-b8d9-1d7544696733",
+        "concepts": ["errors"],
+        "prerequisites": ["strings", "classes-intro"]
```

Comment:
Should each prerequisite match a concept ? I put the `classes-intro` according to the issue associated with this PR

Reply:
Yes!

Reply:
Should I just remove the `classes-intro` prerequisiste then ?

Reply:
No it can stay. It will be a concept :)